src/actions/click-me-modal-open.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Example of an action
+ */
+import { Constants } from "$constants";
+import { ClickMeModal } from "$modals/click-me";
+import { App } from "@slack/bolt";
+
+export class ClickMeModalOpen {
+  app: App;
+
+  constructor(app: App) {
+    this.app = app;
+
+    this.listen();
+  }
+
+  listen() {
+    this.app.action(
+      {
+        type: Constants.BLOCK_ACTIONS,
```

Comment:
Do you mind sharing your thought process about abstracting the type to a constant?

Reply:
No big reason to do so. Often find myself using the 'block_actions' type. To reduce spelling error, I figured it's easier to call BLOCK_ACTIONS from the Constants class.

Reply:
Ok, cool! Yeah, that makes sense, then it's autofilled